src/components/pages/register/forgotPassword/index.tsx

```diff
@@ -136,12 +136,7 @@ export default function ForgotPasswordComponent() {
             </Text>
             <form onSubmit={rForm.handleSubmit(handleOnVerification)}>
               <TextBox name="otp" label={t("OTP.FORM.OTP")} form={rForm} />
-              <TextBox
-                name="password"
-                label={t("USER.PASSWORD")}
-                type="password"
-                form={rForm}
-              />
+              <TextBox name="password" label={t("USER.PASSWORD")} type="password" form={rForm} />
```

Comment:
There seems to be no code change so please revert this file as is

Reply:
vscode code formator onSave automatically align it in the above manner